Source/WebKit/WebProcess/GPU/graphics/RemoteRenderingBackendProxy.cpp

```diff
@@ -153,6 +153,11 @@ RefPtr<ImageBuffer> RemoteRenderingBackendProxy::createImageBuffer(const FloatSi
     return nullptr;
 }
 
+void RemoteRenderingBackendProxy::avoidIOSurfaceSizeCheckInWebProcessForTesting()
```

Comment:
I think this should be named "setAvoid...()" since its sets a flag called "m_avoid...". It can also be inlined in the header file.

Reply:
This is now on HTMLCanvasElement, but I changed the name.